### DISCUSSION TITLE
sbt how to specify a specific local jna ？

### DISCUSSION THREAD

**Discussion by cheungxi** [sbt/sbt#6848] [Q&A]
because upstream jna not support my computer, so I want to use my local jna.

**Comment by eed3si9n** [comment]
Do you mean that you'd like to downgrade or upgrade the JNA version used by sbt itself? I am not sure if that's possible.

**Reply by cheungxi** [reply]
haha, mkdir ./lib in project dir， and move any what you want jna.jar to ./lib  

I have tried, it's ok
